app/src/main/java/com/mobiledevelopmentfoodapp/dto/Food.kt

```diff
@@ -4,7 +4,10 @@ package com.mobiledevelopmentfoodapp.dto
 //import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Food(var name: String, var description: String, var price: Double, @SerializedName("id") var productId:Int = 0 ) {
+data class Food(var name: String,
+                var description: String,
+                var price: Double,
+                @SerializedName("id") var productId:Int = 0 ) {
```

Comment:
Added some line breaks in class parameters for readability per kotlin best practices